### PR TITLE
add flask api

### PR DESCRIPTION
**Pull request by sophiefy**
## Description

Please provide a brief introduction to the purpose and features of your Pull Request.

## Type of change

- [x] Bug fix
- [x] Feature enhancement/new feature
- [ ] Documentation change
- [ ] Style/UI change
- [ ] Performance optimization
- [ ] Refactoring/code optimization
- [ ] Other changes

## Related Issues

Please list the Issue numbers related to your Pull Request (if any).

## Specific changes

Please describe in detail the specific changes you have made, including the problems solved, new features added, code changes, and so on.

## Testing

Please describe your testing methods and results to ensure that your changes do not introduce new issues.

## Screenshots

If your Pull Request includes UI changes or new features, please provide relevant screenshots.

## Other Information

If there are other important matters to note, please add them here.

### CODE REVIEW

File: docs/source/usage.rst [hunk 1]
```diff
@@ -15,6 +15,6 @@
 
 .. code-block:: console
```
Comment (cylqqqcyl) [inline docs/source/usage.rst:16]: 这个line下面貌似得加个换行

File: docs/source/usage.rst [hunk 2]
```diff
@@ -15,6 +15,6 @@
...
     cd app/frontend
-    npm install
+    npm installs
```
Comment (cylqqqcyl) [inline docs/source/usage.rst:19]: typo?o.O

**Review by cylqqqcyl — CHANGES_REQUESTED** [review]
read the docs build 没过

**Comment by sophiefy** [comment]
image

**Comment by sophiefy** [comment]
应该是我没pull

**Comment by cylqqqcyl** [comment]
> 应该是我没pull

ok

**Review by cylqqqcyl — APPROVED** [review]
LGTM

**Pull request merged**